R/utils.R

```diff
@@ -761,3 +761,20 @@ calc_sd <- function(x) {
   }
 }
 
+
+
+#' safe wrapper of Sys.getenv()
+#' 
+#' So far the helper is needed to handle env vars containing `:` 
+#' for which the backslash  is automatically added in some contexts
+#' and R could not get the original value for these env vars.
+#' 
+#' @param x string with the name of the environemntal variable
+#' @param ... additional params for Sys.getenev
+#' @keywords utils
+#
+#' @export 
+#' @return sanitized value of the env variable
+get_env_var <- function(x, ...) {
```

Comment:
Wasn't not saver to leave this function in gDRcomponents?

Reply:
See related discussion here: https://github.com/gdrplatform/gDRutils/pull/136#discussion_r1810510966.